openforcefield/tests/test_forcefield.py

```diff
@@ -1545,6 +1545,50 @@ def test_lookup_parameter_handler_object(self):
         with pytest.raises(NotImplementedError):
             forcefield[type(bonds)]
 
+    def test_hash_cosmetic(self):
+        """Test that adding a cosmetic attribute does not change the hash"""
+        forcefield = ForceField("test_forcefields/smirnoff99Frosst.offxml")
+
+        hash_without_cosmetic = hash(forcefield)
+
+        forcefield.get_parameter_handler("Bonds").add_cosmetic_attribute("foo", 4)
+        assert "foo" in forcefield["Bonds"]._cosmetic_attribs
+
+        hash_with_cosmetic = hash(forcefield)
+
+        assert hash_with_cosmetic == hash_without_cosmetic
+
+    length = 1 * unit.angstrom
+    k = 10 * unit.kilocalorie_per_mole / unit.angstrom ** 2
```

Comment:
Is there a need for these to be class variables? We could move them inside the method below if not.

Reply:
No need, just preference to avoid repeated code. Can be removed if desired (either practical downside or stylistic preference)

Reply:
Ah, on second reading, I can see why this looks weird. At some point I think it would be useful to do this for a bunch of these tests that regularly construct objects with the same values in order to reduce repeated code. But that's out of scope here and doing it for a single test doesn't provide the benefits of doing it everywhere.